commons/c2cgeoportal_commons/__init__.py

```diff
@@ -3,41 +3,25 @@
 
 from pyramid.config import Configurator
 
-from c2cgeoportal_commons.models import (  # noqa
-    get_session_factory,
-    get_engine,
-    get_tm_session,
-    generate_mappers,
-    init_dbsessions,
-)
+import c2cwsgiutils.db
+import c2cwsgiutils.pyramid
+from c2cwsgiutils.health_check import HealthCheck
 
-schema = None  # type: Optional[str]
-srid = None  # type: Optional[str]
+from c2cgeoportal_commons.config import config as configuration
+from c2cgeoportal_commons.models import init_dbsessions
+
+health_check = None
 
 
 def includeme(config: Configurator) -> None:
-    """
-    Initialize the model for a Pyramid app.
-    Activate this setup using ``config.include('c2cgeoportal_admin.commons')``.
-    """
     settings = config.get_settings()
-    settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
-
-    # use pyramid_tm to hook the transaction lifecycle to the request
-    config.include('pyramid_tm')
-
-    # use pyramid_retry to retry a request when transient exceptions occur
-    config.include('pyramid_retry')
-
-    session_factory = get_session_factory(get_engine(settings))
-    config.registry['dbsession_factory'] = session_factory
+    configuration.init(settings.get('app.cfg'))
+    # update the settings object from the YAML application config file
+    settings.update(configuration.get_config())
 
-    # make request.dbsession available for use in Pyramid
-    config.add_request_method(
-        # r.tm is the transaction manager used by pyramid_tm
-        lambda r: get_tm_session(session_factory, r.tm),
-        'dbsession',
-        reify=True
-    )
+    config.include(c2cwsgiutils.pyramid.includeme)
+    global health_check
+    health_check = HealthCheck(config)
```

Comment:
Do you use some c2cwsgiutils functionalities in commons, seems strange to include it here

Reply:
Your completely right, I will remove that...